packages/core/admin/admin/src/pages/ProfilePage/tests/index.test.js

```diff
@@ -5,12 +5,15 @@ import { QueryClient, QueryClientProvider } from 'react-query';
 import { lightTheme, darkTheme } from '@strapi/design-system';
 import ProfilePage from '../index';
 import server from './utils/server';
+import serverLockedSSO from './utils/serverLockedSSO';
 import ThemeToggleProvider from '../../../components/ThemeToggleProvider';
 import Theme from '../../../components/Theme';
 
 jest.mock('../../../components/LocalesProvider/useLocalesProvider', () => () => ({
   changeLocale() {},
-  localeNames: ['en'],
+  localeNames: {
+    en: 'English'
+  },
```

Comment:
If that is needed and the mock doesn't contain it already: could you please extend/ fox the manual mock instead?

Reply:
I removed this part, it isn't needed